test/sequential/sequencerStaking.rules.test.ts

```diff
@@ -353,6 +357,12 @@ describe("sequencerStaking", () => {
           .build(),
       ),
     );
+    const eventFiltered = cancelResolution.filter(
+      (x) => x.method === "L1ReadStored",
+    );
+    expect(eventFiltered[0].event.data[1].toHuman()).toContain(
```

Comment:
plz validate all the fields in the event

Reply:
added